src/api/users/docs/FAQ.md

```diff
@@ -0,0 +1,20 @@
+# FAQ
+
+This document explains several frequently answered questions related to the Users Micro-Service and Firestore. See [`README.md`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/users/README.md) for more information.
+
+### How do we import/export data?
+
+- The good news is that this is possible: the process is more involved that I can (or should) summarize, but it can be found [here](https://firebase.google.com/docs/firestore/manage-data/export-import). The bad news is that importing and exporting of data is directly tied to billing being turned on in the project's account settings.
+  - **Caution**: _Exporting data from Cloud Firestore will incur one read operation per document exported. However, these reads will not appear in the usage section of the console. Make sure you understand this before setting up recurring exports to avoid an unexpected bill._
+
+### How can I add a member to Firestore?
+
+- Navigate to [https://console.firebase.google.com](https://console.firebase.google.com) and select Telescope. Select the gear icon in the upper left-hand corner and click on Users and permissions. From here you can click on the Add members button.
```

Comment:
Is there any precondition for this part? Like should I log in as any account? I visit this address but didn't see an option for telescope to select

Reply:
Ugh sorry, this is not applicable to anyone who doesn't currently own a Firestore db. Currently it's just me, but soon it will be just Dave when we launch our db. I should have explained this better, sorry! Thanks for the feedback though I'll try to make it clearer.

Reply:
oh I see! That makes sense!